writer_test.go

```diff
@@ -1225,6 +1225,27 @@ func ExampleMediaPlaylist_Segments_scte35_oatcls() {
 	// media2.ts
 }
 
+func ExampleMediaPlaylist_Segments_scte35_cue() {
```

Comment:
This function should declare itself a test helper: https://pkg.go.dev/testing#T.Helper

Reply:
Removed this test and created a new one that tests the same functionality in: https://github.com/zencoder/m3u8/pull/8/commits/c953678b1aa9fccf3c8ab6bb7b540872f75ba4c0